src/PresentationalComponents/Modals/AddEditTopic.js

```diff
@@ -0,0 +1,185 @@
+        type === 'DELETE' ?
+            deleteTopic(options) :
+            setTopic(options);
+        handleModalToggle(false);
+    };
+
+    const footer = (
+        <Split gutter='sm' className='split-override'>
+            <SplitItem>
+                <Button key='confirm' variant='primary' onClick={ () => setDeleteHandler({ type: 'SET' })}>
+                    {intl.formatMessage(messages.save)}
+                </Button>
+            </SplitItem>
+            <SplitItem>
+                <Button key='cancel' variant='secondary' onClick={ () => handleModalToggle(false) }>
+                    {intl.formatMessage(messages.cancel)}
+                </Button>
+            </SplitItem>
+            <SplitItem isFilled/>
+            <SplitItem>
+                <Button key='delete' variant='danger' onClick={ () => setDeleteHandler({ type: 'DELETE' })}>
+                    {intl.formatMessage(messages.deleteTopic)}
+                </Button>
+            </SplitItem>
+        </Split>
+    );
+
+    return <Modal
+        title={intl.formatMessage(messages.topicAdminTitle)}
+        isOpen={isModalOpen}
+        onClose={ () => handleModalToggle(false) }
+        footer={footer}
+        isFooterLeftAligned
+        className='modal-width-override'
+    >
+        <Form>
+            <FormGroup
+                label={intl.formatMessage(messages.name)}
+                fieldId='topic-form-name'
+                className='text-input-override'
+            >
+                <TextInput
+                    value={name}
+                    isRequired
+                    type='text'
+                    id='topic-form-name'
+                    name='topic-form-name'
+                    onChange={ (name) => setName(name) }
+                />
+            </FormGroup>
+            <FormGroup
+                label={intl.formatMessage(messages.topicAddEditDescription)}
+                fieldId='topic-form-description'
+                helperText={intl.formatMessage(
+                    messages.topicAddEditDescriptionHelperText
+                )}
+                className='text-area-override'
+            >
+                <TextArea
+                    value={description}
+                    isRequired
+                    name='topic-form-description'
+                    id='topic-form-description'
+                    onChange={ (desc) => setDescription(desc) }
+                />
+            </FormGroup>
+            <FormGroup isInline fieldId='topic-form-labels'>
+                <FormGroup
+                    label={intl.formatMessage(messages.label)}
+                    fieldId='topic-form-tag'
+                    helperText={intl.formatMessage(
+                        messages.topicAddEditTagHelperText
+                    )}
+                    className='text-input-override'
+                >
+                    <TextInput
+                        value={tag}
```

Comment:
maybe do `value={tag.replace(/\s/g,'').toLowerCase()}`

and on 114 ` onChange={ (tag) => setTag(tag.replace(/\s/g,'').toLowerCase()) }`

mocks show label as being without whitespace, gonna also assume lowercase, but here we're just duplicating the string
<img width="672" alt="Screen Shot 2019-12-19 at 11 02 25 AM" src="https://user-images.githubusercontent.com/6640236/71188574-3549a100-224f-11ea-8534-86a0a0932cdf.png">